LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2013 The Apache Software Foundation
+Copyright (c) 2021 The Apache Software Foundation
```

Comment:
please do not remove the 2013 (might have legal consequences)
```suggestion
Copyright (c) 2013 - 2021 The Apache Software Foundation
```

Reply:
Thanks for pointing this @DaanHoogland. I am +1 on `2013 - 2021`.

For the record, Apache Software Foundation recommends such pattern (`Copyright [XXXX-XXXX] The Apache Software Foundation`), as we can see [here](https://www.apache.org/legal/src-headers.html#notice).